ROADMAP.md

```diff
@@ -0,0 +1,105 @@
+# AutoGluon Roadmap 2022
+
+In 2022, our top priorities are:
+
+- [ ] Windows OS Support
+- [ ] Python 3.9 Support
+- [ ] Time-Series Support
+- [ ] HuggingFace Integration
+- [ ] Improved Multi-modal Modeling
+- [ ] Advance SOTA in Stack Ensemble Research (Deeper Stack Ensembles)
+- [ ] Cloud Training & Deployment
+- [ ] Parallel Model Training
```

Comment:
Also worth putting on our list
- Dataset size scaling
- Improved data detectors for preprocessing
Should we discuss factoring out the preprocessor?

Reply:
added:

```
- [ ] Improved Feature Type Inference
- [ ] Improved Feature Preprocessing
- [ ] Improved Large-scale Data Handling (10M+ Rows)
- [ ] Refactor autogluon.features into Standalone Module
```